tests/policies/test_exploration_wrapper.py

```diff
@@ -90,37 +90,65 @@ def test_switch_prob(rng):
     """
     wrapper, venv = make_wrapper(random_prob=0.5, switch_prob=0.0, rng=rng)
     policy = wrapper.current_policy
-    np.random.seed(0)
-    obs = np.random.rand(100, 2)
-    for action in wrapper(obs):
-        assert venv.action_space.contains(action)
-        assert wrapper.current_policy == policy
-
-    def _always_switch(random_prob, num_steps, seed):
-        wrapper, _ = make_wrapper(random_prob=random_prob, switch_prob=1.0, rng=rng)
-        np.random.seed(seed)
-        num_random = 0
-        num_constant = 0
-        for _ in range(num_steps):
-            obs = np.random.rand(1, 2)
-            wrapper(obs)
-            if wrapper.current_policy == wrapper._random_policy:
-                num_random += 1
-            elif wrapper.current_policy == constant_policy:
-                num_constant += 1
-            else:  # pragma: no cover
-                raise ValueError("Unknown policy")
-        return num_random, num_constant
-
-    num_random, num_constant = _always_switch(random_prob=1.0, num_steps=1000, seed=0)
-    assert num_random == 1000
-    assert num_constant == 0
-    num_random, num_constant = _always_switch(random_prob=0.5, num_steps=1000, seed=0)
-    assert num_random > 450
-    assert num_constant > 450
-    num_random, num_constant = _always_switch(random_prob=0.0, num_steps=1000, seed=0)
-    assert num_random == 0
-    assert num_constant == 1000
+
+    num_randoms_prob_one = []
+    num_constants_prob_one = []
+    num_randoms_prob_half = []
+    num_constants_prob_half = []
+    num_randoms_prob_zero = []
+    num_constants_prob_zero = []
+
+    for s in range(5):
```

Comment:
Yeah the specific reason I went for this route was because Windows has a different RNG and the original seed we were using caused the test to fail only on Windows, so this was a way to check that that seed was the actual issue. I think probably the better thing to do here is to assert that the mean is > 450 across the five runs with different seeds.

Reply:
Isn't asserting that the mean is between 450 and 550 mathematically equivalent to having one run that's 5 times as long, and asserting the total is between 2250 and 2750? Admittedly, that's ignoring the unlucky seed issue. 

I also thought that there was a push to try to remove static seed usage in these tests in general.

Reply:
I agree with you, removed dependence on seeds entirely and increased the number of steps.